Detected_Object[] detected_objects_list
